changelog_unreleased/api/pr-7430.md

```diff
@@ -0,0 +1,39 @@
+<!--
```

Comment:
This file is still a TODO. See https://github.com/prettier/prettier/pull/6963#issuecomment-578409884 for context.

Reply:
This is done now. The whole PR is ready for a review. Once this is done, I'm happy to help @fisker with his portion of changelog (see https://github.com/prettier/prettier/pull/6963#issuecomment-578434520).